src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
+    "transport": "http",
+    "icon":"http://google.tieto.com/images/services/google-cloud-platform/BigQuery_500px-1f2fdb56.png",
+    "clients": [
+      {
+      "language":"java",
+      "site":"https://cloud.google.com/bigquery/docs/reference/libraries",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/0.22.0/apidocs/com/google/cloud/bigquery/package-summary.html",
+      "version" : "0.22.0",
+      "description":"The com.google.cloud.bigquery packages",
+      "status" : "beta",  
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigquery",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-bigquery",
+         "version" : "0.22.0-beta"
+      },
+      "platforms" : [
```

Comment:
`Java 7` vs `AES-7` is useful to embed in desktop apps.
`AES-7` vs `AES-8` for Datastore.